src/ocamlorg/lib/package.ml

```diff
@@ -162,6 +166,7 @@ module Info = struct
   let make ~package ~packages ~rev_deps opam =
     let open Lwt.Syntax in
     let+ rev_deps = mk_revdeps package packages rev_deps in
+    let latest = OpamPackage.Version.Map.max_binding version_map in
```

Comment:
Also I'm not confident with this `version_map`

Reply:
This `latest` is unneeded and won't be used so you can delete it.